examples/puppetmaster/smoketest_e2e.sh

```diff
@@ -4,12 +4,12 @@ set -euo pipefail
 
 # Launches a full Puppet stack and converges a node against it
 
-COMPOSE_PROJECT_NAME=puppetmaster
+COMPOSE_PROJECT_NAME=puppetmaster_$(date +"%Y%m%d_%H%M%S")
 
 # make sure on Jenkins if something goes wrong the
 # build doesn't fail because of leftovers from previous tries
 if [ -n "${BUILD_NUMBER:-}" ]; then
-   COMPOSE_PROJECT_NAME=$COMPOSE_PROJECT_NAME-$BUILD_NUMBER
+   COMPOSE_PROJECT_NAME="${COMPOSE_PROJECT_NAME}_${BUILD_NUMBER}"
```

Comment:
why is this even needed if the variable is unique above?

Reply:
Fair point - will fix